rero_mef/api.py

```
@@ -354,35 +469,51 @@ def get_all_ids(cls, with_deleted=False, batch_size=1000, date=None):
+        :raises EntityRecordError.DatabaseRetryError: If database operation fails after retries.
+        """
+        # Retry up to 5 times to handle transient database connection issues
+        for attempt in range(5):
             try:
-                get_count_ok = True
                 return cls._get_all(with_deleted=with_deleted).count()
             except OperationalError:
-                get_count_count += 1
-                msg = f"Get count OperationalError: {get_count_count}"
-                current_app.logger.error(msg)
+                current_app.logger.error(f"Get count OperationalError: {attempt + 1}")
```

Ruff (0.15.1)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)